index.js

```diff
@@ -101,7 +104,7 @@ function processKeyValuePair(key, value, parent, theirs, mine, path, options = {
       // If single dimension array run the comparison
       if (literalValues.length == value.length) {
         path.push(key);
-        var differences = compareValues(parent[key], theirs[key], mine[key], path, options[key]);
+        var differences = compareValues(parent[key], theirs[key], mine[key], path, options[key] || {});
```

Comment:
options[key] may not exist, I'll have to add the default at the start of this function.